It stops a lot for no reason and needs memory clear sometimes several times to restart

[ Environment ]
Android version: 9
OS version: 4.4.153-perf-ge135357f77c3
AntennaPod version: 1.8.1
Model: moto x4
Device: payton_sprout
Product: payton_fi

[ Crash info ]
Time: 19-05-2020 21:33:02
AntennaPod version: 1.8.1

[ StackTrace ]
```
io.reactivex.exceptions.OnErrorNotImplementedException: The exception was not handled due to missing onError handler in the subscribe() method call. Further reading: https://github.com/ReactiveX/RxJava/wiki/Error-Handling | Protocol has invalid length (25 at position 0): [25, -63, -105, -126, -27, 42, 88, 51, -80]
	at io.reactivex.internal.observers.CallbackCompletableObserver.accept(SourceFile:47)
	at io.reactivex.internal.observers.CallbackCompletableObserver.accept(SourceFile:26)
	at io.reactivex.internal.observers.CallbackCompletableObserver.onError(SourceFile:64)
	at io.reactivex.internal.operators.completable.CompletableObserveOn$ObserveOnCompletableObserver.run(SourceFile:87)
	at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(SourceFile:119)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6912)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:860)
Caused by: java.lang.IllegalArgumentException: Protocol has invalid length (25 at position 0): [25, -63, -105, -126, -27, 42, 88, 51, -80]
	at com.google.android.gms.org.conscrypt.SSLUtils.decodeProtocols(:com.google.android.gms@201516030@20.15.16 (100408-309763488):3)
	at com.google.android.gms.org.conscrypt.OpenSSLSocketImpl.setAlpnProtocols(:com.google.android.gms@201516030@20.15.16 (100408-309763488):1)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.okhttp.internal.OptionalMethod.invoke(OptionalMethod.java:112)
	at com.android.okhttp.internal.OptionalMethod.invokeWithoutCheckedException(OptionalMethod.java:130)
	at com.android.okhttp.internal.Platform.configureTlsExtensions(Platform.java:118)
	at com.android.okhttp.internal.io.RealConnection.connectTls(RealConnection.java:187)
	at com.android.okhttp.internal.io.RealConnection.connectSocket(RealConnection.java:149)
	at com.android.okhttp.internal.io.RealConnection.connect(RealConnection.java:112)
	at com.android.okhttp.internal.http.StreamAllocation.findConnection(StreamAllocation.java:184)
	at com.android.okhttp.internal.http.StreamAllocation.findHealthyConnection(StreamAllocation.java:126)
	at com.android.okhttp.internal.http.StreamAllocation.newStream(StreamAllocation.java:95)
	at com.android.okhttp.internal.http.HttpEngine.connect(HttpEngine.java:281)
	at com.android.okhttp.internal.http.HttpEngine.sendRequest(HttpEngine.java:224)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:461)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:407)
	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:244)
	at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:210)
	at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:26)
	at java.net.URL.openStream(URL.java:1072)
	at de.danoeh.antennapod.core.util.ChapterUtils.readOggChaptersFromPlayableStreamUrl(SourceFile:158)
	at de.danoeh.antennapod.core.util.ChapterUtils.loadChaptersFromStreamUrl(SourceFile:60)
	at de.danoeh.antennapod.core.feed.FeedMedia.loadChapterMarks(SourceFile:401)
	at de.danoeh.antennapod.core.service.playback.PlaybackServiceTaskManager.lambda$startChapterLoader$1(SourceFile:313)
	at de.danoeh.antennapod.core.service.playback.-$$Lambda$PlaybackServiceTaskManager$JGTJkWwv5gEUn75t4vz00Lpz_5o.subscribe(Unknown Source:2)
	at io.reactivex.internal.operators.completable.CompletableCreate.subscribeActual(SourceFile:39)
	at io.reactivex.Completable.subscribe(SourceFile:2185)
	at io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run(SourceFile:64)
	at io.reactivex.Scheduler$DisposeTask.run(SourceFile:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(SourceFile:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(SourceFile:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)
```

--- a/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceTaskManager.java
+++ b/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceTaskManager.java
@@ -311,12 +311,13 @@ public class PlaybackServiceTaskManager {
 
         if (media.getChapters() == null) {
             Completable.create(emitter -> {
-                        media.loadChapterMarks();
-                        emitter.onComplete();
-                    })
+                media.loadChapterMarks();
+                emitter.onComplete();
+            })
                     .subscribeOn(Schedulers.io())
                     .observeOn(AndroidSchedulers.mainThread())
-                    .subscribe(() -> callback.onChapterLoaded(media));
+                    .subscribe(() -> callback.onChapterLoaded(media),
+                            throwable -> Log.d(TAG, "Error loading chapters: " + Log.getStackTraceString(throwable)));
         }
     }
 

